[2016-12-04 11:34:37] DEBUG [django.db.backends:90] (0.030) 
            SELECT name, type FROM sqlite_master
            WHERE type in ('table', 'view') AND NOT name='sqlite_sequence'
            ORDER BY name; args=None
[2016-12-04 11:34:37] DEBUG [django.db.backends:90] (0.001) SELECT "django_migrations"."app", "django_migrations"."name" FROM "django_migrations"; args=()
[2016-12-04 13:09:40] DEBUG [django.db.backends:90] (0.014) SELECT "django_session"."session_key", "django_session"."session_data", "django_session"."expire_date" FROM "django_session" WHERE ("django_session"."expire_date" > '2016-12-04 18:09:40.172692' AND "django_session"."session_key" = 'ctyrbouy3oohdousc8py71pl91qwpht5'); args=('2016-12-04 18:09:40.172692', 'ctyrbouy3oohdousc8py71pl91qwpht5')
[2016-12-04 13:09:40] DEBUG [django.db.backends:90] (0.000) SELECT "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by" FROM "auths_user" WHERE "auths_user"."id" = '01e659bc77a5436f9fc859d23a77e6aa'; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-12-04 13:09:40] DEBUG [django.template:929] Exception while resolving variable 'is_popup' in template 'admin/index.html'.
Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\context.py", line 75, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 891, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in "[{'False': False, 'None': None, 'True': True}, {'request': <WSGIRequest: GET '/admin/'>, 'DEFAULT_MESSAGE_LEVELS': {'INFO': 20, 'SUCCESS': 25, 'DEBUG': 10, 'WARNING': 30, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x042D4490>, 'user': <SimpleLazyObject: <User: admin>>, 'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x042D56A8>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x04E01210>}, {}, {'app_list': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0356E6F0>, 'app_url': '/admin/auth/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0455F370>, 'object_name': 'Group', 'admin_url': '/admin/auth/group/', 'add_url': '/admin/auth/group/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x042D4190>, 'object_name': 'Permission', 'admin_url': '/admin/auth/permission/', 'add_url': '/admin/auth/permission/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'auth'}, {'name': 'Auths', 'app_url': '/admin/auths/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x045981B0>, 'object_name': 'HierarchyType', 'admin_url': '/admin/auths/hierarchytype/', 'add_url': '/admin/auths/hierarchytype/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598110>, 'object_name': 'Hierarchy', 'admin_url': '/admin/auths/hierarchy/', 'add_url': '/admin/auths/hierarchy/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598390>, 'object_name': 'Menu', 'admin_url': '/admin/auths/menu/', 'add_url': '/admin/auths/menu/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0455F330>, 'object_name': 'Person', 'admin_url': '/admin/auths/person/', 'add_url': '/admin/auths/person/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x045981D0>, 'object_name': 'UserHierarchyGroup', 'admin_url': '/admin/auths/userhierarchygroup/', 'add_url': '/admin/auths/userhierarchygroup/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598190>, 'object_name': 'UserHierarchyPermission', 'admin_url': '/admin/auths/userhierarchypermission/', 'add_url': '/admin/auths/userhierarchypermission/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598150>, 'object_name': 'User', 'admin_url': '/admin/auths/user/', 'add_url': '/admin/auths/user/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'auths'}, {'name': 'Django OAuth Toolkit', 'app_url': '/admin/oauth2_provider/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598810>, 'object_name': 'AccessToken', 'admin_url': '/admin/oauth2_provider/accesstoken/', 'add_url': '/admin/oauth2_provider/accesstoken/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598450>, 'object_name': 'Application', 'admin_url': '/admin/oauth2_provider/application/', 'add_url': '/admin/oauth2_provider/application/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x045982F0>, 'object_name': 'Grant', 'admin_url': '/admin/oauth2_provider/grant/', 'add_url': '/admin/oauth2_provider/grant/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0455F350>, 'object_name': 'RefreshToken', 'admin_url': '/admin/oauth2_provider/refreshtoken/', 'add_url': '/admin/oauth2_provider/refreshtoken/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'oauth2_provider'}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0356E830>, 'app_url': '/admin/contenttypes/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x042D4150>, 'object_name': 'ContentType', 'admin_url': '/admin/contenttypes/contenttype/', 'add_url': '/admin/contenttypes/contenttype/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'contenttypes'}, {'name': 'Topico2', 'app_url': '/admin/topico2/', 'has_module_perms': True, 'models': [{'name': 'Antecedentemedicos', 'object_name': 'Antecedentemedico', 'admin_url': '/admin/topico2/antecedentemedico/', 'add_url': '/admin/topico2/antecedentemedico/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Consultas', 'object_name': 'Consulta', 'admin_url': '/admin/topico2/consulta/', 'add_url': '/admin/topico2/consulta/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Departamentos', 'object_name': 'Departamento', 'admin_url': '/admin/topico2/departamento/', 'add_url': '/admin/topico2/departamento/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Derivacions', 'object_name': 'Derivacion', 'admin_url': '/admin/topico2/derivacion/', 'add_url': '/admin/topico2/derivacion/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Diagnosticos', 'object_name': 'Diagnostico', 'admin_url': '/admin/topico2/diagnostico/', 'add_url': '/admin/topico2/diagnostico/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'EspecificacionRecetas', 'object_name': 'EspecificacionReceta', 'admin_url': '/admin/topico2/especificacionreceta/', 'add_url': '/admin/topico2/especificacionreceta/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Funcionvitals', 'object_name': 'Funcionvital', 'admin_url': '/admin/topico2/funcionvital/', 'add_url': '/admin/topico2/funcionvital/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Historias', 'object_name': 'Historia', 'admin_url': '/admin/topico2/historia/', 'add_url': '/admin/topico2/historia/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Medicamentos', 'object_name': 'Medicamento', 'admin_url': '/admin/topico2/medicamento/', 'add_url': '/admin/topico2/medicamento/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Pacientes', 'object_name': 'Paciente', 'admin_url': '/admin/topico2/paciente/', 'add_url': '/admin/topico2/paciente/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Tratamientos', 'object_name': 'Tratamiento', 'admin_url': '/admin/topico2/tratamiento/', 'add_url': '/admin/topico2/tratamiento/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'topico2'}], 'site_url': '/', 'title': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0355ACD0>, 'site_title': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0355AC10>, 'LANGUAGE_BIDI': False, 'site_header': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0355AC70>, 'has_permission': True, 'LANGUAGE_CODE': 'es', 'available_apps': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0356E6F0>, 'app_url': '/admin/auth/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C9B0>, 'object_name': 'Group', 'admin_url': '/admin/auth/group/', 'add_url': '/admin/auth/group/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458CD70>, 'object_name': 'Permission', 'admin_url': '/admin/auth/permission/', 'add_url': '/admin/auth/permission/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'auth'}, {'name': 'Auths', 'app_url': '/admin/auths/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C050>, 'object_name': 'HierarchyType', 'admin_url': '/admin/auths/hierarchytype/', 'add_url': '/admin/auths/hierarchytype/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C850>, 'object_name': 'Hierarchy', 'admin_url': '/admin/auths/hierarchy/', 'add_url': '/admin/auths/hierarchy/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C430>, 'object_name': 'Menu', 'admin_url': '/admin/auths/menu/', 'add_url': '/admin/auths/menu/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458CD50>, 'object_name': 'Person', 'admin_url': '/admin/auths/person/', 'add_url': '/admin/auths/person/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C230>, 'object_name': 'UserHierarchyGroup', 'admin_url': '/admin/auths/userhierarchygroup/', 'add_url': '/admin/auths/userhierarchygroup/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C0D0>, 'object_name': 'UserHierarchyPermission', 'admin_url': '/admin/auths/userhierarchypermission/', 'add_url': '/admin/auths/userhierarchypermission/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C410>, 'object_name': 'User', 'admin_url': '/admin/auths/user/', 'add_url': '/admin/auths/user/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'auths'}, {'name': 'Django OAuth Toolkit', 'app_url': '/admin/oauth2_provider/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C590>, 'object_name': 'AccessToken', 'admin_url': '/admin/oauth2_provider/accesstoken/', 'add_url': '/admin/oauth2_provider/accesstoken/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458CF50>, 'object_name': 'Application', 'admin_url': '/admin/oauth2_provider/application/', 'add_url': '/admin/oauth2_provider/application/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C5D0>, 'object_name': 'Grant', 'admin_url': '/admin/oauth2_provider/grant/', 'add_url': '/admin/oauth2_provider/grant/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C7F0>, 'object_name': 'RefreshToken', 'admin_url': '/admin/oauth2_provider/refreshtoken/', 'add_url': '/admin/oauth2_provider/refreshtoken/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'oauth2_provider'}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0356E830>, 'app_url': '/admin/contenttypes/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598410>, 'object_name': 'ContentType', 'admin_url': '/admin/contenttypes/contenttype/', 'add_url': '/admin/contenttypes/contenttype/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'contenttypes'}, {'name': 'Topico2', 'app_url': '/admin/topico2/', 'has_module_perms': True, 'models': [{'name': 'Antecedentemedicos', 'object_name': 'Antecedentemedico', 'admin_url': '/admin/topico2/antecedentemedico/', 'add_url': '/admin/topico2/antecedentemedico/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Consultas', 'object_name': 'Consulta', 'admin_url': '/admin/topico2/consulta/', 'add_url': '/admin/topico2/consulta/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Departamentos', 'object_name': 'Departamento', 'admin_url': '/admin/topico2/departamento/', 'add_url': '/admin/topico2/departamento/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Derivacions', 'object_name': 'Derivacion', 'admin_url': '/admin/topico2/derivacion/', 'add_url': '/admin/topico2/derivacion/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Diagnosticos', 'object_name': 'Diagnostico', 'admin_url': '/admin/topico2/diagnostico/', 'add_url': '/admin/topico2/diagnostico/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'EspecificacionRecetas', 'object_name': 'EspecificacionReceta', 'admin_url': '/admin/topico2/especificacionreceta/', 'add_url': '/admin/topico2/especificacionreceta/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Funcionvitals', 'object_name': 'Funcionvital', 'admin_url': '/admin/topico2/funcionvital/', 'add_url': '/admin/topico2/funcionvital/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Historias', 'object_name': 'Historia', 'admin_url': '/admin/topico2/historia/', 'add_url': '/admin/topico2/historia/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Medicamentos', 'object_name': 'Medicamento', 'admin_url': '/admin/topico2/medicamento/', 'add_url': '/admin/topico2/medicamento/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Pacientes', 'object_name': 'Paciente', 'admin_url': '/admin/topico2/paciente/', 'add_url': '/admin/topico2/paciente/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Tratamientos', 'object_name': 'Tratamiento', 'admin_url': '/admin/topico2/tratamiento/', 'add_url': '/admin/topico2/tratamiento/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'topico2'}]}]"
[2016-12-04 13:09:40] DEBUG [django.template:929] Exception while resolving variable 'is_popup' in template 'admin/index.html'.
Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\context.py", line 75, in __getitem__
    raise KeyError(key)
KeyError: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 891, in _resolve_lookup
    if isinstance(current, BaseContext) and getattr(type(current), bit):
AttributeError: type object 'RequestContext' has no attribute 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'is_popup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [is_popup] in "[{'False': False, 'None': None, 'True': True}, {'request': <WSGIRequest: GET '/admin/'>, 'DEFAULT_MESSAGE_LEVELS': {'INFO': 20, 'SUCCESS': 25, 'DEBUG': 10, 'WARNING': 30, 'ERROR': 40}, 'messages': <django.contrib.messages.storage.fallback.FallbackStorage object at 0x042D4490>, 'user': <SimpleLazyObject: <User: admin>>, 'csrf_token': <SimpleLazyObject: <function csrf.<locals>._get_val at 0x042D56A8>>, 'perms': <django.contrib.auth.context_processors.PermWrapper object at 0x04E01210>}, {}, {'app_list': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0356E6F0>, 'app_url': '/admin/auth/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0455F370>, 'object_name': 'Group', 'admin_url': '/admin/auth/group/', 'add_url': '/admin/auth/group/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x042D4190>, 'object_name': 'Permission', 'admin_url': '/admin/auth/permission/', 'add_url': '/admin/auth/permission/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'auth'}, {'name': 'Auths', 'app_url': '/admin/auths/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x045981B0>, 'object_name': 'HierarchyType', 'admin_url': '/admin/auths/hierarchytype/', 'add_url': '/admin/auths/hierarchytype/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598110>, 'object_name': 'Hierarchy', 'admin_url': '/admin/auths/hierarchy/', 'add_url': '/admin/auths/hierarchy/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598390>, 'object_name': 'Menu', 'admin_url': '/admin/auths/menu/', 'add_url': '/admin/auths/menu/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0455F330>, 'object_name': 'Person', 'admin_url': '/admin/auths/person/', 'add_url': '/admin/auths/person/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x045981D0>, 'object_name': 'UserHierarchyGroup', 'admin_url': '/admin/auths/userhierarchygroup/', 'add_url': '/admin/auths/userhierarchygroup/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598190>, 'object_name': 'UserHierarchyPermission', 'admin_url': '/admin/auths/userhierarchypermission/', 'add_url': '/admin/auths/userhierarchypermission/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598150>, 'object_name': 'User', 'admin_url': '/admin/auths/user/', 'add_url': '/admin/auths/user/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'auths'}, {'name': 'Django OAuth Toolkit', 'app_url': '/admin/oauth2_provider/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598810>, 'object_name': 'AccessToken', 'admin_url': '/admin/oauth2_provider/accesstoken/', 'add_url': '/admin/oauth2_provider/accesstoken/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598450>, 'object_name': 'Application', 'admin_url': '/admin/oauth2_provider/application/', 'add_url': '/admin/oauth2_provider/application/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x045982F0>, 'object_name': 'Grant', 'admin_url': '/admin/oauth2_provider/grant/', 'add_url': '/admin/oauth2_provider/grant/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0455F350>, 'object_name': 'RefreshToken', 'admin_url': '/admin/oauth2_provider/refreshtoken/', 'add_url': '/admin/oauth2_provider/refreshtoken/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'oauth2_provider'}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0356E830>, 'app_url': '/admin/contenttypes/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x042D4150>, 'object_name': 'ContentType', 'admin_url': '/admin/contenttypes/contenttype/', 'add_url': '/admin/contenttypes/contenttype/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'contenttypes'}, {'name': 'Topico2', 'app_url': '/admin/topico2/', 'has_module_perms': True, 'models': [{'name': 'Antecedentemedicos', 'object_name': 'Antecedentemedico', 'admin_url': '/admin/topico2/antecedentemedico/', 'add_url': '/admin/topico2/antecedentemedico/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Consultas', 'object_name': 'Consulta', 'admin_url': '/admin/topico2/consulta/', 'add_url': '/admin/topico2/consulta/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Departamentos', 'object_name': 'Departamento', 'admin_url': '/admin/topico2/departamento/', 'add_url': '/admin/topico2/departamento/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Derivacions', 'object_name': 'Derivacion', 'admin_url': '/admin/topico2/derivacion/', 'add_url': '/admin/topico2/derivacion/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Diagnosticos', 'object_name': 'Diagnostico', 'admin_url': '/admin/topico2/diagnostico/', 'add_url': '/admin/topico2/diagnostico/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'EspecificacionRecetas', 'object_name': 'EspecificacionReceta', 'admin_url': '/admin/topico2/especificacionreceta/', 'add_url': '/admin/topico2/especificacionreceta/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Funcionvitals', 'object_name': 'Funcionvital', 'admin_url': '/admin/topico2/funcionvital/', 'add_url': '/admin/topico2/funcionvital/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Historias', 'object_name': 'Historia', 'admin_url': '/admin/topico2/historia/', 'add_url': '/admin/topico2/historia/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Medicamentos', 'object_name': 'Medicamento', 'admin_url': '/admin/topico2/medicamento/', 'add_url': '/admin/topico2/medicamento/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Pacientes', 'object_name': 'Paciente', 'admin_url': '/admin/topico2/paciente/', 'add_url': '/admin/topico2/paciente/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Tratamientos', 'object_name': 'Tratamiento', 'admin_url': '/admin/topico2/tratamiento/', 'add_url': '/admin/topico2/tratamiento/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'topico2'}], 'site_url': '/', 'title': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0355ACD0>, 'site_title': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0355AC10>, 'LANGUAGE_BIDI': False, 'site_header': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0355AC70>, 'has_permission': True, 'LANGUAGE_CODE': 'es', 'available_apps': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0356E6F0>, 'app_url': '/admin/auth/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C9B0>, 'object_name': 'Group', 'admin_url': '/admin/auth/group/', 'add_url': '/admin/auth/group/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458CD70>, 'object_name': 'Permission', 'admin_url': '/admin/auth/permission/', 'add_url': '/admin/auth/permission/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'auth'}, {'name': 'Auths', 'app_url': '/admin/auths/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C050>, 'object_name': 'HierarchyType', 'admin_url': '/admin/auths/hierarchytype/', 'add_url': '/admin/auths/hierarchytype/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C850>, 'object_name': 'Hierarchy', 'admin_url': '/admin/auths/hierarchy/', 'add_url': '/admin/auths/hierarchy/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C430>, 'object_name': 'Menu', 'admin_url': '/admin/auths/menu/', 'add_url': '/admin/auths/menu/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458CD50>, 'object_name': 'Person', 'admin_url': '/admin/auths/person/', 'add_url': '/admin/auths/person/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C230>, 'object_name': 'UserHierarchyGroup', 'admin_url': '/admin/auths/userhierarchygroup/', 'add_url': '/admin/auths/userhierarchygroup/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C0D0>, 'object_name': 'UserHierarchyPermission', 'admin_url': '/admin/auths/userhierarchypermission/', 'add_url': '/admin/auths/userhierarchypermission/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C410>, 'object_name': 'User', 'admin_url': '/admin/auths/user/', 'add_url': '/admin/auths/user/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'auths'}, {'name': 'Django OAuth Toolkit', 'app_url': '/admin/oauth2_provider/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C590>, 'object_name': 'AccessToken', 'admin_url': '/admin/oauth2_provider/accesstoken/', 'add_url': '/admin/oauth2_provider/accesstoken/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458CF50>, 'object_name': 'Application', 'admin_url': '/admin/oauth2_provider/application/', 'add_url': '/admin/oauth2_provider/application/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C5D0>, 'object_name': 'Grant', 'admin_url': '/admin/oauth2_provider/grant/', 'add_url': '/admin/oauth2_provider/grant/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0458C7F0>, 'object_name': 'RefreshToken', 'admin_url': '/admin/oauth2_provider/refreshtoken/', 'add_url': '/admin/oauth2_provider/refreshtoken/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'oauth2_provider'}, {'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x0356E830>, 'app_url': '/admin/contenttypes/', 'has_module_perms': True, 'models': [{'name': <django.utils.functional.lazy.<locals>.__proxy__ object at 0x04598410>, 'object_name': 'ContentType', 'admin_url': '/admin/contenttypes/contenttype/', 'add_url': '/admin/contenttypes/contenttype/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'contenttypes'}, {'name': 'Topico2', 'app_url': '/admin/topico2/', 'has_module_perms': True, 'models': [{'name': 'Antecedentemedicos', 'object_name': 'Antecedentemedico', 'admin_url': '/admin/topico2/antecedentemedico/', 'add_url': '/admin/topico2/antecedentemedico/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Consultas', 'object_name': 'Consulta', 'admin_url': '/admin/topico2/consulta/', 'add_url': '/admin/topico2/consulta/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Departamentos', 'object_name': 'Departamento', 'admin_url': '/admin/topico2/departamento/', 'add_url': '/admin/topico2/departamento/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Derivacions', 'object_name': 'Derivacion', 'admin_url': '/admin/topico2/derivacion/', 'add_url': '/admin/topico2/derivacion/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Diagnosticos', 'object_name': 'Diagnostico', 'admin_url': '/admin/topico2/diagnostico/', 'add_url': '/admin/topico2/diagnostico/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'EspecificacionRecetas', 'object_name': 'EspecificacionReceta', 'admin_url': '/admin/topico2/especificacionreceta/', 'add_url': '/admin/topico2/especificacionreceta/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Funcionvitals', 'object_name': 'Funcionvital', 'admin_url': '/admin/topico2/funcionvital/', 'add_url': '/admin/topico2/funcionvital/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Historias', 'object_name': 'Historia', 'admin_url': '/admin/topico2/historia/', 'add_url': '/admin/topico2/historia/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Medicamentos', 'object_name': 'Medicamento', 'admin_url': '/admin/topico2/medicamento/', 'add_url': '/admin/topico2/medicamento/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Pacientes', 'object_name': 'Paciente', 'admin_url': '/admin/topico2/paciente/', 'add_url': '/admin/topico2/paciente/add/', 'perms': {'add': True, 'change': True, 'delete': True}}, {'name': 'Tratamientos', 'object_name': 'Tratamiento', 'admin_url': '/admin/topico2/tratamiento/', 'add_url': '/admin/topico2/tratamiento/add/', 'perms': {'add': True, 'change': True, 'delete': True}}], 'app_label': 'topico2'}]}]"
[2016-12-04 13:09:40] DEBUG [django.db.backends:90] (0.012) SELECT "django_admin_log"."id", "django_admin_log"."action_time", "django_admin_log"."user_id", "django_admin_log"."content_type_id", "django_admin_log"."object_id", "django_admin_log"."object_repr", "django_admin_log"."action_flag", "django_admin_log"."change_message", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "django_content_type"."id", "django_content_type"."app_label", "django_content_type"."model" FROM "django_admin_log" INNER JOIN "auths_user" ON ("django_admin_log"."user_id" = "auths_user"."id") LEFT OUTER JOIN "django_content_type" ON ("django_admin_log"."content_type_id" = "django_content_type"."id") WHERE "django_admin_log"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa' ORDER BY "django_admin_log"."action_time" DESC LIMIT 10; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-12-04 13:09:40] INFO [django.server:131] "GET /admin/ HTTP/1.1" 200 17348
[2016-12-04 13:09:40] INFO [django.server:131] "GET /static/admin/css/base.css HTTP/1.1" 304 0
[2016-12-04 13:09:40] INFO [django.server:131] "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
[2016-12-04 13:09:40] INFO [django.server:131] "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
[2016-12-04 13:09:40] INFO [django.server:131] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
[2016-12-04 13:09:40] INFO [django.server:131] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
[2016-12-04 13:09:40] INFO [django.server:131] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
[2016-12-04 13:09:40] INFO [django.server:131] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
[2016-12-04 13:09:40] INFO [django.server:131] "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
[2016-12-04 13:09:40] DEBUG [django.template:929] Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\urls\resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\django\\contrib\\admindocs\\urls.py'> (None:None) ^admin/doc/>], [<RegexURLResolver <module 'rest_framework.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\rest_framework\\urls.py'> (rest_framework:rest_framework) ^api-auth/>], [<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\oauth2_provider\\urls.py'> (None:oauth2_provider) ^o/>], [<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\pru\\topico\\medicfastv2_service\\medicfastv2_service_apps\\auths_api\\urls.py'> (None:None) ^api/auths/>], [<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\pru\\topico\\medicfastv2_service\\medicfastv2_service_apps\\topico2_api\\urls.py'> (None:None) ^api/topico2/>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in '<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
[2016-12-04 13:09:40] DEBUG [django.template:929] Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\urls\resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\django\\contrib\\admindocs\\urls.py'> (None:None) ^admin/doc/>], [<RegexURLResolver <module 'rest_framework.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\rest_framework\\urls.py'> (rest_framework:rest_framework) ^api-auth/>], [<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\oauth2_provider\\urls.py'> (None:oauth2_provider) ^o/>], [<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\pru\\topico\\medicfastv2_service\\medicfastv2_service_apps\\auths_api\\urls.py'> (None:None) ^api/auths/>], [<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\pru\\topico\\medicfastv2_service\\medicfastv2_service_apps\\topico2_api\\urls.py'> (None:None) ^api/topico2/>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\\\pru\\\\topico\\\\ve_medicfast\\\\lib\\\\site-packages\\\\django\\\\contrib\\\\admindocs\\\\urls.py'> (None:None) ^admin/doc/>"
[2016-12-04 13:09:40] DEBUG [django.template:929] Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\urls\resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\django\\contrib\\admindocs\\urls.py'> (None:None) ^admin/doc/>], [<RegexURLResolver <module 'rest_framework.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\rest_framework\\urls.py'> (rest_framework:rest_framework) ^api-auth/>], [<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\oauth2_provider\\urls.py'> (None:oauth2_provider) ^o/>], [<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\pru\\topico\\medicfastv2_service\\medicfastv2_service_apps\\auths_api\\urls.py'> (None:None) ^api/auths/>], [<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\pru\\topico\\medicfastv2_service\\medicfastv2_service_apps\\topico2_api\\urls.py'> (None:None) ^api/topico2/>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'rest_framework.urls' from 'D:\\\\pru\\\\topico\\\\ve_medicfast\\\\lib\\\\site-packages\\\\rest_framework\\\\urls.py'> (rest_framework:rest_framework) ^api-auth/>"
[2016-12-04 13:09:40] DEBUG [django.template:929] Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\urls\resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\django\\contrib\\admindocs\\urls.py'> (None:None) ^admin/doc/>], [<RegexURLResolver <module 'rest_framework.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\rest_framework\\urls.py'> (rest_framework:rest_framework) ^api-auth/>], [<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\oauth2_provider\\urls.py'> (None:oauth2_provider) ^o/>], [<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\pru\\topico\\medicfastv2_service\\medicfastv2_service_apps\\auths_api\\urls.py'> (None:None) ^api/auths/>], [<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\pru\\topico\\medicfastv2_service\\medicfastv2_service_apps\\topico2_api\\urls.py'> (None:None) ^api/topico2/>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\\\pru\\\\topico\\\\ve_medicfast\\\\lib\\\\site-packages\\\\oauth2_provider\\\\urls.py'> (None:oauth2_provider) ^o/>"
[2016-12-04 13:09:40] DEBUG [django.template:929] Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\urls\resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\django\\contrib\\admindocs\\urls.py'> (None:None) ^admin/doc/>], [<RegexURLResolver <module 'rest_framework.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\rest_framework\\urls.py'> (rest_framework:rest_framework) ^api-auth/>], [<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\oauth2_provider\\urls.py'> (None:oauth2_provider) ^o/>], [<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\pru\\topico\\medicfastv2_service\\medicfastv2_service_apps\\auths_api\\urls.py'> (None:None) ^api/auths/>], [<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\pru\\topico\\medicfastv2_service\\medicfastv2_service_apps\\topico2_api\\urls.py'> (None:None) ^api/topico2/>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\\\pru\\\\topico\\\\medicfastv2_service\\\\medicfastv2_service_apps\\\\auths_api\\\\urls.py'> (None:None) ^api/auths/>"
[2016-12-04 13:09:40] DEBUG [django.template:929] Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\core\handlers\base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\urls\resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'django.contrib.admindocs.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\django\\contrib\\admindocs\\urls.py'> (None:None) ^admin/doc/>], [<RegexURLResolver <module 'rest_framework.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\rest_framework\\urls.py'> (rest_framework:rest_framework) ^api-auth/>], [<RegexURLResolver <module 'oauth2_provider.urls' from 'D:\\pru\\topico\\ve_medicfast\\lib\\site-packages\\oauth2_provider\\urls.py'> (None:oauth2_provider) ^o/>], [<RegexURLResolver <module 'medicfastv2_service_apps.auths_api.urls' from 'D:\\pru\\topico\\medicfastv2_service\\medicfastv2_service_apps\\auths_api\\urls.py'> (None:None) ^api/auths/>], [<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\pru\\topico\\medicfastv2_service\\medicfastv2_service_apps\\topico2_api\\urls.py'> (None:None) ^api/topico2/>]], 'path': 'favicon.ico'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\pru\topico\ve_medicfast\lib\site-packages\django\template\base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'medicfastv2_service_apps.topico2_api.urls' from 'D:\\\\pru\\\\topico\\\\medicfastv2_service\\\\medicfastv2_service_apps\\\\topico2_api\\\\urls.py'> (None:None) ^api/topico2/>"
[2016-12-04 13:09:40] WARNING [django.request:152] Not Found: /favicon.ico
[2016-12-04 13:09:40] WARNING [django.server:131] "GET /favicon.ico HTTP/1.1" 404 2494
[2016-12-04 13:09:49] INFO [django.server:131] "OPTIONS /api/auths/localuserinfo/ HTTP/1.1" 200 0
[2016-12-04 13:09:49] INFO [django.server:131] "OPTIONS /api/auths/usermenu/ HTTP/1.1" 200 0
[2016-12-04 13:09:49] DEBUG [django.db.backends:90] (0.016) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'TJs1tVIXGvUQF45QbJNTD9OUuyD1LD'; args=('TJs1tVIXGvUQF45QbJNTD9OUuyD1LD',)
[2016-12-04 13:09:49] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'TJs1tVIXGvUQF45QbJNTD9OUuyD1LD'; args=('TJs1tVIXGvUQF45QbJNTD9OUuyD1LD',)
[2016-12-04 13:09:49] DEBUG [django.db.backends:90] (0.021) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'TJs1tVIXGvUQF45QbJNTD9OUuyD1LD'; args=('TJs1tVIXGvUQF45QbJNTD9OUuyD1LD',)
[2016-12-04 13:09:49] DEBUG [django.db.backends:90] (0.000) SELECT "auth_group"."id", "auth_group"."name" FROM "auth_group" INNER JOIN "auths_user_groups" ON ("auth_group"."id" = "auths_user_groups"."group_id") WHERE "auths_user_groups"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa'; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-12-04 13:09:49] DEBUG [django.db.backends:90] (0.010) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'TJs1tVIXGvUQF45QbJNTD9OUuyD1LD'; args=('TJs1tVIXGvUQF45QbJNTD9OUuyD1LD',)
[2016-12-04 13:09:49] DEBUG [django.db.backends:90] (0.001) SELECT "auth_permission"."id", "auth_permission"."name", "auth_permission"."content_type_id", "auth_permission"."codename" FROM "auth_permission" INNER JOIN "auths_user_user_permissions" ON ("auth_permission"."id" = "auths_user_user_permissions"."permission_id") INNER JOIN "django_content_type" ON ("auth_permission"."content_type_id" = "django_content_type"."id") WHERE "auths_user_user_permissions"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa' ORDER BY "django_content_type"."app_label" ASC, "django_content_type"."model" ASC, "auth_permission"."codename" ASC; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-12-04 13:09:49] INFO [django.server:131] "GET /api/auths/localuserinfo/ HTTP/1.1" 200 405
[2016-12-04 13:09:49] DEBUG [django.db.backends:90] (0.000) SELECT "auths_menu"."id" FROM "auths_menu" WHERE ("auths_menu"."id" IS NOT NULL AND "auths_menu"."module" = 'BACKEND' AND "auths_menu"."is_active" = 1) ORDER BY "auths_menu"."pos" ASC; args=('BACKEND', True)
[2016-12-04 13:09:49] DEBUG [django.db.backends:90] (0.000) SELECT (1) AS "a" FROM "django_session" WHERE "django_session"."session_key" = 'ehky844xfbqj35mseet5z9aoqqsajcev' LIMIT 1; args=('ehky844xfbqj35mseet5z9aoqqsajcev',)
[2016-12-04 13:09:49] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-12-04 13:09:49] DEBUG [django.db.backends:90] (0.013) INSERT INTO "django_session" ("session_key", "session_data", "expire_date") SELECT 'ehky844xfbqj35mseet5z9aoqqsajcev', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-12-04 18:10:19.425564'; args=('ehky844xfbqj35mseet5z9aoqqsajcev', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-12-04 18:10:19.425564')
[2016-12-04 13:09:49] INFO [django.server:131] "GET /api/auths/usermenu/ HTTP/1.1" 200 11
[2016-12-04 13:10:00] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 13:10:00] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 13:10:09] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_application" WHERE "oauth2_provider_application"."client_id" = 'XHJWk31Fzj0uLmx49TZ8zJ1Dti31HAUTKrPNuf65'; args=('XHJWk31Fzj0uLmx49TZ8zJ1Dti31HAUTKrPNuf65',)
[2016-12-04 13:10:09] DEBUG [django.db.backends:90] (0.000) SELECT "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by" FROM "auths_user" WHERE "auths_user"."username" = 'admin'; args=('admin',)
[2016-12-04 13:10:09] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-12-04 13:10:09] DEBUG [django.db.backends:90] (0.001) INSERT INTO "oauth2_provider_accesstoken" ("user_id", "token", "application_id", "expires", "scope") VALUES ('01e659bc77a5436f9fc859d23a77e6aa', 'kz43lG4U8zGg9FLFtntSLSJEMXNCS1', 1, '2016-12-05 04:10:09.432051', 'write read'); args=['01e659bc77a5436f9fc859d23a77e6aa', 'kz43lG4U8zGg9FLFtntSLSJEMXNCS1', 1, '2016-12-05 04:10:09.432051', 'write read']
[2016-12-04 13:10:09] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-12-04 13:10:09] DEBUG [django.db.backends:90] (0.006) INSERT INTO "oauth2_provider_refreshtoken" ("user_id", "token", "application_id", "access_token_id") VALUES ('01e659bc77a5436f9fc859d23a77e6aa', 'sGeq0qACn9TFREentt6us4Y00RhVzU', 1, 25); args=['01e659bc77a5436f9fc859d23a77e6aa', 'sGeq0qACn9TFREentt6us4Y00RhVzU', 1, 25]
[2016-12-04 13:10:09] INFO [django.server:131] "POST /o/token/ HTTP/1.1" 200 169
[2016-12-04 13:10:10] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'kz43lG4U8zGg9FLFtntSLSJEMXNCS1'; args=('kz43lG4U8zGg9FLFtntSLSJEMXNCS1',)
[2016-12-04 13:10:10] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'kz43lG4U8zGg9FLFtntSLSJEMXNCS1'; args=('kz43lG4U8zGg9FLFtntSLSJEMXNCS1',)
[2016-12-04 13:10:10] DEBUG [django.db.backends:90] (0.011) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'kz43lG4U8zGg9FLFtntSLSJEMXNCS1'; args=('kz43lG4U8zGg9FLFtntSLSJEMXNCS1',)
[2016-12-04 13:10:10] DEBUG [django.db.backends:90] (0.004) SELECT "auth_group"."id", "auth_group"."name" FROM "auth_group" INNER JOIN "auths_user_groups" ON ("auth_group"."id" = "auths_user_groups"."group_id") WHERE "auths_user_groups"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa'; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-12-04 13:10:10] DEBUG [django.db.backends:90] (0.000) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'kz43lG4U8zGg9FLFtntSLSJEMXNCS1'; args=('kz43lG4U8zGg9FLFtntSLSJEMXNCS1',)
[2016-12-04 13:10:10] DEBUG [django.db.backends:90] (0.001) SELECT "auth_permission"."id", "auth_permission"."name", "auth_permission"."content_type_id", "auth_permission"."codename" FROM "auth_permission" INNER JOIN "auths_user_user_permissions" ON ("auth_permission"."id" = "auths_user_user_permissions"."permission_id") INNER JOIN "django_content_type" ON ("auth_permission"."content_type_id" = "django_content_type"."id") WHERE "auths_user_user_permissions"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa' ORDER BY "django_content_type"."app_label" ASC, "django_content_type"."model" ASC, "auth_permission"."codename" ASC; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-12-04 13:10:10] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."id" FROM "auths_menu" WHERE ("auths_menu"."id" IS NOT NULL AND "auths_menu"."module" = 'BACKEND' AND "auths_menu"."is_active" = 1) ORDER BY "auths_menu"."pos" ASC; args=('BACKEND', True)
[2016-12-04 13:10:10] INFO [django.server:131] "GET /api/auths/localuserinfo/ HTTP/1.1" 200 405
[2016-12-04 13:10:10] DEBUG [django.db.backends:90] (0.027) SELECT (1) AS "a" FROM "django_session" WHERE "django_session"."session_key" = 'pvkmz77numw5kzlib8ikt0d37za7hzuj' LIMIT 1; args=('pvkmz77numw5kzlib8ikt0d37za7hzuj',)
[2016-12-04 13:10:10] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-12-04 13:10:10] DEBUG [django.db.backends:90] (0.001) INSERT INTO "django_session" ("session_key", "session_data", "expire_date") SELECT 'pvkmz77numw5kzlib8ikt0d37za7hzuj', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-12-04 18:10:40.627666'; args=('pvkmz77numw5kzlib8ikt0d37za7hzuj', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-12-04 18:10:40.627666')
[2016-12-04 13:10:11] INFO [django.server:131] "GET /api/auths/usermenu/ HTTP/1.1" 200 11
[2016-12-04 16:22:16] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 16:22:16] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 16:22:55] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 16:22:55] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 16:23:05] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 16:23:05] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 16:23:39] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 16:23:39] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 16:25:17] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 16:25:17] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 17:00:39] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 17:00:39] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:43:34] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:43:34] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:43:37] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:43:37] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:51:21] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:51:21] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:51:55] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:51:55] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:52:15] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:52:15] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:52:20] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:52:20] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:52:21] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:52:21] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:52:33] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:52:33] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:53:32] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:53:32] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:54:10] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:54:10] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:54:33] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:54:33] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:54:59] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:54:59] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:55:08] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:55:08] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:57:04] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:57:04] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:57:14] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:57:14] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:57:37] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:57:37] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:57:48] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:57:48] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:58:01] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:58:01] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 18:58:12] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 18:58:12] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:02:53] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:02:53] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:03:12] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:03:12] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:03:25] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:03:25] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:03:55] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:03:55] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:04:09] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:04:09] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:06:55] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:06:55] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:14:14] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:14:14] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:16:06] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:16:06] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:16:09] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:16:09] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:17:10] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:17:10] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:17:11] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:17:11] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:17:12] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:17:12] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:17:37] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:17:37] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:17:56] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:17:56] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:18:12] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:18:12] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:20:57] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:20:57] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:37:25] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:37:25] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:37:45] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:37:45] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:38:21] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:38:21] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:39:02] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:39:02] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:39:32] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:39:32] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:39:42] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:39:42] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:39:49] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:39:49] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:40:06] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:40:06] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:40:54] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:40:54] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:41:08] DEBUG [django.db.backends:90] (0.003) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:41:08] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:41:17] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:41:17] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:41:28] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:41:28] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:41:35] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:41:35] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:41:44] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:41:44] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:41:53] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:41:53] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:41:59] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:41:59] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:42:14] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:42:14] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:42:26] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:42:26] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:42:36] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:42:36] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:42:45] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:42:45] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:42:59] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:42:59] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:43:16] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:43:16] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:43:20] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:43:20] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:43:57] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:43:57] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:44:08] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:44:08] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:44:18] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:44:18] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:44:29] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:44:29] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:44:38] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:44:38] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:44:51] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:44:51] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:45:04] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:45:04] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:45:28] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:45:28] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:45:41] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:45:41] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:45:53] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:45:53] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:46:13] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:46:13] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:47:19] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:47:19] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:47:38] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:47:38] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:47:53] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:47:53] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:47:54] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:47:54] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:48:09] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:48:09] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:48:28] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:48:28] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:48:48] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:48:48] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:49:32] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:49:32] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:50:00] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:50:00] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:50:09] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_application" WHERE "oauth2_provider_application"."client_id" = 'XHJWk31Fzj0uLmx49TZ8zJ1Dti31HAUTKrPNuf65'; args=('XHJWk31Fzj0uLmx49TZ8zJ1Dti31HAUTKrPNuf65',)
[2016-12-04 19:50:09] DEBUG [django.db.backends:90] (0.001) SELECT "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by" FROM "auths_user" WHERE "auths_user"."username" = 'admin'; args=('admin',)
[2016-12-04 19:50:09] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-12-04 19:50:09] DEBUG [django.db.backends:90] (0.001) INSERT INTO "oauth2_provider_accesstoken" ("user_id", "token", "application_id", "expires", "scope") VALUES ('01e659bc77a5436f9fc859d23a77e6aa', 'kY9CD1CKqzYKci2cngdHYWe2wgN3nk', 1, '2016-12-05 10:50:09.316323', 'write read'); args=['01e659bc77a5436f9fc859d23a77e6aa', 'kY9CD1CKqzYKci2cngdHYWe2wgN3nk', 1, '2016-12-05 10:50:09.316323', 'write read']
[2016-12-04 19:50:09] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-12-04 19:50:09] DEBUG [django.db.backends:90] (0.001) INSERT INTO "oauth2_provider_refreshtoken" ("user_id", "token", "application_id", "access_token_id") VALUES ('01e659bc77a5436f9fc859d23a77e6aa', 'BrNdVs8PucotjmawvwreT9rsBnD5VL', 1, 26); args=['01e659bc77a5436f9fc859d23a77e6aa', 'BrNdVs8PucotjmawvwreT9rsBnD5VL', 1, 26]
[2016-12-04 19:50:09] INFO [django.server:131] "POST /o/token/ HTTP/1.1" 200 169
[2016-12-04 19:50:10] DEBUG [django.db.backends:90] (0.002) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'kY9CD1CKqzYKci2cngdHYWe2wgN3nk'; args=('kY9CD1CKqzYKci2cngdHYWe2wgN3nk',)
[2016-12-04 19:50:10] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'kY9CD1CKqzYKci2cngdHYWe2wgN3nk'; args=('kY9CD1CKqzYKci2cngdHYWe2wgN3nk',)
[2016-12-04 19:50:10] DEBUG [django.db.backends:90] (0.006) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'kY9CD1CKqzYKci2cngdHYWe2wgN3nk'; args=('kY9CD1CKqzYKci2cngdHYWe2wgN3nk',)
[2016-12-04 19:50:10] DEBUG [django.db.backends:90] (0.001) SELECT "oauth2_provider_accesstoken"."id", "oauth2_provider_accesstoken"."user_id", "oauth2_provider_accesstoken"."token", "oauth2_provider_accesstoken"."application_id", "oauth2_provider_accesstoken"."expires", "oauth2_provider_accesstoken"."scope", "auths_user"."password", "auths_user"."last_login", "auths_user"."is_superuser", "auths_user"."username", "auths_user"."first_name", "auths_user"."last_name", "auths_user"."email", "auths_user"."is_staff", "auths_user"."is_active", "auths_user"."date_joined", "auths_user"."id", "auths_user"."last_hierarchy_id", "auths_user"."last_module_id", "auths_user"."person_id", "auths_user"."updated_at", "auths_user"."registered_by", "oauth2_provider_application"."id", "oauth2_provider_application"."client_id", "oauth2_provider_application"."user_id", "oauth2_provider_application"."redirect_uris", "oauth2_provider_application"."client_type", "oauth2_provider_application"."authorization_grant_type", "oauth2_provider_application"."client_secret", "oauth2_provider_application"."name", "oauth2_provider_application"."skip_authorization" FROM "oauth2_provider_accesstoken" LEFT OUTER JOIN "auths_user" ON ("oauth2_provider_accesstoken"."user_id" = "auths_user"."id") INNER JOIN "oauth2_provider_application" ON ("oauth2_provider_accesstoken"."application_id" = "oauth2_provider_application"."id") WHERE "oauth2_provider_accesstoken"."token" = 'kY9CD1CKqzYKci2cngdHYWe2wgN3nk'; args=('kY9CD1CKqzYKci2cngdHYWe2wgN3nk',)
[2016-12-04 19:50:10] DEBUG [django.db.backends:90] (0.005) SELECT "auths_menu"."id" FROM "auths_menu" WHERE ("auths_menu"."id" IS NOT NULL AND "auths_menu"."module" = 'BACKEND' AND "auths_menu"."is_active" = 1) ORDER BY "auths_menu"."pos" ASC; args=('BACKEND', True)
[2016-12-04 19:50:10] DEBUG [django.db.backends:90] (0.000) SELECT "auth_group"."id", "auth_group"."name" FROM "auth_group" INNER JOIN "auths_user_groups" ON ("auth_group"."id" = "auths_user_groups"."group_id") WHERE "auths_user_groups"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa'; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-12-04 19:50:10] DEBUG [django.db.backends:90] (0.000) SELECT "auth_permission"."id", "auth_permission"."name", "auth_permission"."content_type_id", "auth_permission"."codename" FROM "auth_permission" INNER JOIN "auths_user_user_permissions" ON ("auth_permission"."id" = "auths_user_user_permissions"."permission_id") INNER JOIN "django_content_type" ON ("auth_permission"."content_type_id" = "django_content_type"."id") WHERE "auths_user_user_permissions"."user_id" = '01e659bc77a5436f9fc859d23a77e6aa' ORDER BY "django_content_type"."app_label" ASC, "django_content_type"."model" ASC, "auth_permission"."codename" ASC; args=('01e659bc77a5436f9fc859d23a77e6aa',)
[2016-12-04 19:50:10] INFO [django.server:131] "GET /api/auths/localuserinfo/ HTTP/1.1" 200 405
[2016-12-04 19:50:10] DEBUG [django.db.backends:90] (0.060) SELECT (1) AS "a" FROM "django_session" WHERE "django_session"."session_key" = 'aazzxwb8qw9nyq8x54m9gj364evkd2i4' LIMIT 1; args=('aazzxwb8qw9nyq8x54m9gj364evkd2i4',)
[2016-12-04 19:50:10] DEBUG [django.db.backends:90] (0.000) BEGIN; args=None
[2016-12-04 19:50:10] DEBUG [django.db.backends:90] (0.001) INSERT INTO "django_session" ("session_key", "session_data", "expire_date") SELECT 'aazzxwb8qw9nyq8x54m9gj364evkd2i4', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-12-05 00:50:40.378803'; args=('aazzxwb8qw9nyq8x54m9gj364evkd2i4', 'ZWMyMmE2MTA1MzM3NDcxZTRmYzk5YzliZTNiYjg2ZTZkMjFlMTc0NTp7Il9zZXNzaW9uX2V4cGlyeSI6MzB9', '2016-12-05 00:50:40.378803')
[2016-12-04 19:50:10] INFO [django.server:131] "GET /api/auths/usermenu/ HTTP/1.1" 200 11
[2016-12-04 19:50:16] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:50:16] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:50:39] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:50:39] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:50:52] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:50:52] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:51:14] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:51:14] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:52:00] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:52:00] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:52:16] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:52:16] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:53:06] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:53:06] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:53:22] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:53:22] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:53:42] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:53:42] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 19:54:01] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 19:54:01] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:04:35] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:04:35] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:05:36] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:05:36] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:07:05] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:07:05] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:07:20] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:07:20] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:08:03] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:08:03] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:08:15] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:08:15] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:08:29] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:08:29] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:08:30] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:08:30] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:08:48] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:08:48] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:09:17] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:09:17] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:09:42] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:09:42] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:09:55] DEBUG [django.db.backends:90] (0.002) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:09:55] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:10:08] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:10:08] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:10:28] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:10:28] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:10:41] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:10:41] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:11:25] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:11:25] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:12:06] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:12:06] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:12:19] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:12:19] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:12:20] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:12:20] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
[2016-12-04 20:13:08] DEBUG [django.db.backends:90] (0.001) SELECT "auths_menu"."router_json" FROM "auths_menu" ORDER BY "auths_menu"."pos" ASC; args=()
[2016-12-04 20:13:08] INFO [django.server:131] "GET /api/auths/routers/ HTTP/1.1" 200 2
